util/cache/cache.go

```diff
@@ -129,6 +139,62 @@ func getFlagVal[T any](cmd *cobra.Command, o Options, name string, getVal func(n
 	}
 }
 
+// loadRedisCredsFromFile loads Redis credentials from a file mounted at the specified path.
+func loadRedisCredsFromSecret(mountPath string, opt Options) *redisCreds {
+	creds := &redisCreds{}
+	if mountPath != "" {
+		log.Info("Loading Redis credentials from file: ", mountPath)
+		readAuthDetailsFromFile := func(filename string) string {
+			data, err := os.ReadFile(filepath.Join(mountPath, filename))
+			if err != nil {
+				log.Infof("Could not read Redis credential file %s: %v. Falling back to environment variable if available.", filepath.Join(mountPath, filename), err)
```

Comment:
Should be a warn since the mountPath has been configured

Reply:
I do feel like the behaviour should be to warn if the filePath is specified, then file has an error, BUT the env variable has a value.
If the filePath is specified, then file has an error, AND the env variable is also empty, then it sounds like maybe it is a real error and we should fail.